repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java

```diff
@@ -846,6 +859,54 @@ public String createVertexIndex(AtlasGraphManagement management, String property
         return indexFieldName;
     }
 
+    public String createVertexIndex(AtlasAttributeDef attributeDef, AtlasGraphManagement management, String propertyName, UniqueKind uniqueKind, Class propertyClass,
```

Comment:
I see the code is duplicated here, you can even call base method from this overloaded method, without duplicating the code.

Reply:
I've created a new method `createVertexCompositeIndex` to reduce code duplication.